x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/420_connector_sync_job_check_in.yml

```diff
@@ -2,6 +2,7 @@ setup:
   - skip:
       version: " - 8.11.99"
       reason: Introduced in 8.12.0
+      features: is_after
```

Comment:
I'm curious why do we need to include this in the `skip` section?

Reply:
I had to declare the new `IsAfterAssertion` as a "Feature", so that test runners can skip tests, if they don't support this feature.